Mathlib/Analysis/NormedSpace/lpSpace.lean

```diff
@@ -1217,3 +1220,43 @@ instance completeSpace : CompleteSpace (lp E p) :=
 end Topology
 
 end lp
+
+section Lipschitz
+
+local notation "ℓ^∞(" ι ")" => lp (fun i : ι => ℝ) ∞
```

Comment:
Now that you have scoped notation, you should be able to do `open scoped lp` to get the notation back in this context without needing to restate a `notation` like this.